src/main/java/uk/ac/ebi/eva/bd2k/export/StudyExporter.java

```diff
@@ -0,0 +1,51 @@
+package uk.ac.ebi.eva.bd2k.export;
+
+import uk.ac.ebi.ddi.xml.validator.parser.marshaller.OmicsDataMarshaller;
+import uk.ac.ebi.ddi.xml.validator.parser.model.Database;
+
+import uk.ac.ebi.eva.bd2k.client.StudyClient;
+import uk.ac.ebi.eva.bd2k.transform.StudyTransformer;
+import uk.ac.ebi.eva.lib.models.VariantStudy;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+import java.time.LocalDate;
+import java.util.Collections;
+
+public class StudyExporter {
+
+    private final StudyClient studyClient;
+
+    private final StudyTransformer transformer;
+
+    private final OmicsDataMarshaller marshaller;
+
+    private String outputDirectory;
+
+    public StudyExporter(StudyClient studyClient, StudyTransformer transformer, OmicsDataMarshaller marshaller, String outputDirectory) {
+        this.studyClient = studyClient;
+        this.transformer = transformer;
+        this.marshaller = marshaller;
+        this.outputDirectory = outputDirectory;
+    }
+
+    public void export() throws FileNotFoundException {
+        for (VariantStudy study : studyClient.getAllStudies()) {
+            OutputStream os = new FileOutputStream(outputDirectory + "/" + study.getId() + ".xml");
+            Database database = buildDatabase(study);
+            marshaller.marshall(database, os);
+        }
+    }
+
+    // TODO: extract to class
+    private Database buildDatabase(VariantStudy study) {
+        Database database = new Database();
+        // TODO: review those fields
+        database.setName("EVA");
+        database.setRelease("BETA");
```

Comment:
I've done this change in 40e1a00, but now we have some redundancy in the produced XML:

```<database>
  <name>EVA</name>
  <release>2017-05-17</release>
  <release_date>2017-05-17</release_date>
  <entries>
      ...
```
Both elements are mandatory according to the specification, so if we don't have another release value candidate, we can leave it like this.